### DISCUSSION TITLE
科幻蟲洞可能存在，量子電腦首次模擬蟲洞工作機制

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#882] [General]
蟲洞的觀念 在我們現時生活中不斷的被討論 而時空旅行也僅僅在電影中呈現 希望有朝一日能夢想成真

https://tw.news.yahoo.com/%E7%A7%91%E5%B9%BB%E8%9F%B2%E6%B4%9E%E5%8F%AF%E8%83%BD%E5%AD%98%E5%9C%A8-%E9%87%8F%E5%AD%90%E9%9B%BB%E8%85%A6%E9%A6%96%E6%AC%A1%E6%A8%A1%E6%93%AC%E8%9F%B2%E6%B4%9E%E5%B7%A5%E4%BD%9C%E6%A9%9F%E5%88%B6-020512058.html

**Comment by StevenWanglolz** [comment]
這篇文章裡面提到了好多艱深的專業用詞啊~實在是有看沒有懂，不過希望經過這次的模擬能幫助天文學家在宇宙中找到蟲洞，然後我們也許就能找到瞬間移動或時空旅行的方法了哈哈哈

**Comment by Chiway1229** [comment]
雖然我不是很懂，不過感覺這種東西就是很酷
也許有一天能在這個領域有突破也說不定!

**Comment by bonny123865** [comment]
蟲洞這個東西感覺好玄，而且引發一個蟲洞感覺也不是一般設備能做到的事情。
希望科學家能在天文方面做突破，搞不好就可以有很多未知的能力了（都在亂幻想

**Comment by Evan0979** [comment]
蟲洞到底是否真的存在啊?宇宙真的有好多未知的奧秘

**Comment by CPCpyt** [comment]
雖然我有看沒有懂，對我來說真的太艱澀了...
不過我想對於有能之士是一個很棒的構想和啟發吧。
